__snapshots__/Toast.md

```diff
@@ -0,0 +1,136 @@
+# `Toast`
+
+#### `loads`
+
+```html
+<sp-icons-medium style="display: none;"></sp-icons-medium>
+<div class="body">
+    <div class="content">
+        <slot></slot>
+    </div>
+    <slot name="action"></slot>
+</div>
+<div class="buttons">
+    <sp-clear-button
+        data-js-focus-visible=""
+        label="Close"
+        tabindex="0"
+        variant="overBackground"
+    ></sp-clear-button>
+</div>
+```
+
+#### `loads - [variant="negative"]`
+
+```html
+<sp-icons-medium style="display: none;"></sp-icons-medium>
+<sp-icon class="type" label="Negative" name="ui:AlertSmall" size="s"></sp-icon>
+<div class="body">
+    <div class="content">
+        <slot></slot>
+    </div>
+    <slot name="action"></slot>
+</div>
+<div class="buttons">
+    <sp-clear-button
+        data-js-focus-visible=""
+        label="Close"
+        tabindex="0"
+        variant="overBackground"
+    ></sp-clear-button>
+</div>
+```
+
+#### `loads - [variant="positive"]`
+
+```html
+<sp-icons-medium style="display: none;"></sp-icons-medium>
+<sp-icon
+    class="type"
+    label="Positive"
+    name="ui:SuccessSmall"
+    size="s"
+></sp-icon>
+<div class="body">
+    <div class="content">
+        <slot></slot>
+    </div>
+    <slot name="action"></slot>
+</div>
+<div class="buttons">
+    <sp-clear-button
+        data-js-focus-visible=""
+        label="Close"
+        tabindex="0"
+        variant="overBackground"
+    ></sp-clear-button>
+</div>
+```
+
+#### `loads - [variant="info"]`
+
+```html
+<sp-icons-medium style="display: none;"></sp-icons-medium>
+<sp-icon
+    class="type"
+    label="Information"
+    name="ui:InfoSmall"
+    size="s"
+></sp-icon>
+<div class="body">
+    <div class="content">
+        <slot></slot>
+    </div>
+    <slot name="action"></slot>
+</div>
+<div class="buttons">
+    <sp-clear-button
+        data-js-focus-visible=""
+        label="Close"
+        tabindex="0"
+        variant="overBackground"
+    ></sp-clear-button>
+</div>
+```
+
+#### `loads - [variant="error"]`
```

Comment:
should we bother exposing the error/warning variants on the storybook page? it looks like the spectrum-css advises using the negative variant instead.
https://opensource.adobe.com/spectrum-css/components/toast/#Error

Reply:
I could see allowing `error` as a pseudonym for `negative`. I'm guessing most coders are gonna *want* to call the red version an error.

I'd also vote for keeping warning since it'll help with our adoption (but we should plan to list it as deprecated).

Reply:
sounds reasonable to leave it until spectrum-css removes the variants 👍 and just to clarify, do we need to leave it in stories for testing coverage reasons?

Reply:
It doesn't need to remain in the stories, no. I'll remove there.